HLeithner/joomla-cms
# Prepared mod whosonline

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Use the module in all ways you can think of.

* create a frontend module
* set the filter by group
* log into frontend with a non admin user
* You should see all logged in users with the access level your user have access to.

### Expected result
Nothing changed.
